### PR TITLE
Mergeblas

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

**Comment by btracey** [comment]
This does the same actions as the main `gonum` PR. Merges the appropriate repos, rewrites the imports to be for `gonum.org/v1/gonum `, and deletes the travis files. It does one extra thing, which is to move the BLAS travis files to the main repository so we have somewhere to start when we try to get it to pass next time.

File: blas/blas32/blas32.go [hunk 1]
```diff
@@ -6,8 +6,8 @@
 package blas32
 
 import (
-	"github.com/gonum/blas"
-	"github.com/gonum/blas/native"
+	"gonum.org/v1/gonum/blas"
+	"gonum.org/v1/gonum/blas/native"
```
Comment (kortschak) [inline blas/blas32/blas32.go]: "gonum.org/v1/netlib/blas/native"...

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+sudo: required
+
+language: go
+
```
Comment (kortschak) [inline .travis.yml]: You will need a line

```
go_import_path: gonum.org/v1/netlib
```

File: blas/blas32/blas32.go (hunk 1 above)
Comment (btracey) [inline blas/blas32/blas32.go]: But that's not right, is it? The pure go implementations are staying in `gonum`. Either way, the first PR will be deleting all but the `cgo` packages so it doesn't really matter.

Comment (kortschak) [inline blas/blas32/blas32.go]: Sorry, yes, you are right.

**Comment by kortschak** [comment]
Presumably you need to do this for this PR:

```
git mv blas/cgo/ cblas
git mv lapack/cgo/ lapacke
git rm -r lapack
git rm -r blas
```

**Comment by kortschak** [comment]
Also, I see that the generation code will be broken without merging the internal repo for the binding package. That is only used for generating C bindings, so it fits here.

**Comment by btracey** [comment]
Closing this and trying again so that `internal` doesn't hide the successive commits.

**Comment by kortschak** [comment]
I'm working on a complete PR for this - hopefully to pass travis.

**Comment by btracey** [comment]
Oh, should I stop then?

**Comment by kortschak** [comment]
Yes please, give me a little while.

**Pull request closed without merging**